auto_landing/src/landing.cpp

```diff
@@ -0,0 +1,17 @@
+#include<land_uav/landing.hpp>
+
+namespace ariitk::land_uav;
```

Comment:
Everything function definition should be inside curly braces of namespace. However you have terminated it with a semi colon.